free-2.md

```diff
@@ -58,12 +72,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+A significant amount of time would have to be spent researching people's needs and figuring out how to turn those into coherent, fluent commands. Inspiration could be taken from python's existing library to see what sort of data manipulation is possible and how they are carried out. 
```

Comment:
Interacting with Excel either requires using the existing microsoft .Net assemblies, or writing a parser/generator for the syntax of the language. Both of those end up being rather painful.